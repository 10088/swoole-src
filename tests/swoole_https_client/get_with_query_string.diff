001+ Fatal error: Uncaught Error: Call to undefined function socket_create() in /src/swoole-src/tests/include/toolkit/functions.php:219
001- SUCCESS
002+ Stack trace:
003+ #0 /src/swoole-src/tests/include/toolkit/functions.php(557): get_one_free_port()
004+ #1 /src/swoole-src/tests/swoole_https_client/get_with_query_string.php(7): ServerManager->listen('127.0.0.1')
005+ #2 {main}
006+   thrown in /src/swoole-src/tests/include/toolkit/functions.php on line 219