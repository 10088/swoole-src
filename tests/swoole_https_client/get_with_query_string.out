Fatal error: Uncaught Error: Call to undefined function socket_create() in /src/swoole-src/tests/include/toolkit/functions.php:219
Stack trace:
#0 /src/swoole-src/tests/include/toolkit/functions.php(557): get_one_free_port()
#1 /src/swoole-src/tests/swoole_https_client/get_with_query_string.php(7): ServerManager->listen('127.0.0.1')
#2 {main}
  thrown in /src/swoole-src/tests/include/toolkit/functions.php on line 219